cs/cs/VowpalWabbitMultiLine.cs

```diff
@@ -424,7 +424,7 @@ public static class VowpalWabbitMultiLine
             int i = 0;
             foreach (var index in values)
             {
-                result[i++] = validActionDependentFeatures[index];
+                result[i++] = validActionDependentFeatures[(int)index.Action];
```

Comment:
This is the new action/score feature John implemented. This is 0-based. Only exploration library has 1-based actions. VW prediction output was always 0-based.

Markus